ci/linux-debian.Dockerfile

```diff
@@ -11,21 +11,25 @@ RUN dpkg --add-architecture i386 && \
 RUN apt-get update && apt-get install --no-install-recommends -y \
         git ca-certificates \
         make automake libtool pkg-config dpkg-dev valgrind qemu-user \
-        gcc clang llvm libc6-dbg \
+        gcc clang llvm libclang-rt-dev libc6-dbg \
         g++ \
-        gcc-i686-linux-gnu libc6-dev-i386-cross libc6-dbg:i386 libubsan1:i386 libasan6:i386 \
+        gcc-i686-linux-gnu libc6-dev-i386-cross libc6-dbg:i386 libubsan1:i386 libasan8:i386 \
         gcc-s390x-linux-gnu libc6-dev-s390x-cross libc6-dbg:s390x \
         gcc-arm-linux-gnueabihf libc6-dev-armhf-cross libc6-dbg:armhf \
         gcc-aarch64-linux-gnu libc6-dev-arm64-cross libc6-dbg:arm64 \
         gcc-powerpc64le-linux-gnu libc6-dev-ppc64el-cross libc6-dbg:ppc64el \
-        gcc-mingw-w64-x86-64-win32 wine64 wine \
+        gcc-mingw-w64-x86-64-win32 wine64 \
         gcc-mingw-w64-i686-win32 wine32 \
         sagemath
 
 WORKDIR /root
+
+# A workaround for `wine` package failure to employ the Debian alternatives system properly.
+RUN ln -s /usr/lib/wine/wine64 /usr/bin/wine64
```

Comment:
What happens if we don't include this?

Reply:
The `wine64` won't be available in `PATH`, and the [`wine-msvc.sh`](https://github.com/mstorsjo/msvc-wine/blob/master/wrappers/wine-msvc.sh) script will fail.

Reply:
Ok, I see, wine-msvc.sh has `wine64` hardcoded. Then I suggest including this `ln` in the `RUN` command of wine-msvc below. (Unless we really want to get rid of the `wine` convenience package, then we'll always need some manual symlink.)

Reply:
> Then I suggest including this `ln` in the `RUN` command of wine-msvc below.

Done.